packages/store/src/legacyMap.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2019 Agoric, under Apache license 2.0
+
```

Comment:
```suggestion
// @ts-check
```

And in a few other files as well

Reply:
Thanks for catching this! I don't see them anymore. I deleted all those is store/ . Resolving, but there's still a lot in the rest of our code. We should delete those separately.